Style and broken picture

piaudonn opened the pull request:
Section: Create a Web App and App Service Plan
- Added numbered list to stick to the style of the rest of the page.
- Added missing punctuation.
- Fixed broken picture (the raw code was displayed instead of the actual picture).

azurecla commented:
Hi __@raymondlaghaeian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

piaudonn committed:
Style and broken picture

Section: Create a Web App and App Service Plan
- Added numbered list to stick to the style of the rest of the page.
- Added missing punctuation.
- Fixed broken picture (the raw code was displayed instead of the actual picture).
